lib/utils/oidc.js

```diff
@@ -158,9 +167,6 @@ async function oidc ({ packageName, registry, opts, config }) {
     log.silly('oidc', `OIDC token successfully retrieved`)
   } catch (error) {
     log.verbose('oidc', error.message)
```

Comment:
I believe we're avoiding using `log.error` because "OIDC is always an optional feature". Can we pass an error object to `log.verbose`, same as `log.error`?

When logging an error, I've found it useful to have a static error message you can search the code for, informing you of where the error was caught and logged.  

```suggestion
    log.verbose('oidc', 'Error preparing for OIDC publish', error)
```